Makefile

```
@@ -58,6 +58,13 @@ examples_install:
 	${MKDIR} ${SCRIPTS_INSTALL_PATH}/examples/dnsdoctor
 	${INSTALL} -m 0644 examples/dnsdoctor/*.lua ${SCRIPTS_INSTALL_PATH}/examples/dnsdoctor
 
+ipparse_install:
```

I would call this `moonstatik_install`
```suggestion
moonstatik_install:
```

---

The fact is it doesn’t install whole moontastik, but only ipparse, which is a lib, so intended to be useful for other scripts that could be part of lunatik's main repo. Moontastik contains also (for now) [hexdump](https://github.com/luainkernel/moontastik/tree/master/hexdump), which doesn’t get installed here.

---

my point is to decouple lunatik from moonstatik.. that is, lunatik base doesn't need to have the visibility of moonstatik modules.. we can install "moonstatik" from here and let moonstatik base decide whatever will be present in the default package